cylc/flow/task_job_mgr.py

```diff
@@ -834,15 +869,12 @@ def _prep_submit_task_job_error(self, suite, itask, action, exc):
 
     def _prep_submit_task_job_impl(self, suite, itask, rtconfig):
         """Helper for self._prep_submit_task_job."""
-        itask.task_owner = rtconfig['remote']['owner']
-        if itask.task_owner:
-            owner_at_host = itask.task_owner + "@" + itask.task_host
-        else:
-            owner_at_host = itask.task_host
-        itask.summary['host'] = owner_at_host
-        itask.summary['job_hosts'][itask.submit_num] = owner_at_host
+        # itask.platform is going to get boring...
+        platform = itask.platform
```

Comment:
I thought it would be easier than consistently referencing `itask.platform`. I don't have a strong attachment to this code one way or the other, although I think that your way makes the code read better as part of the whole system (cut out the step of finding out that platform = itask.platform when tracing a bug), so I'll change it. 'tis trivial anyhow.